static/css/timetable/partials/course_modal.scss

```diff
@@ -71,6 +71,7 @@
   align-items: center;
   margin: 6px;
   >.areas {
+    margin-right: 2%;
```

Comment:
I think you did this for the search results bubbles but it messes up the course modal bubbles. Let's make a new selector for search results bubbles and put this in there as well as make the bubbles smaller. Look at how I selected the bubbles in exp modal as a reference

Reply:
Might even be able to just use the exp modal sizes